R/Simulations-class.R

```diff
@@ -1,4 +1,3 @@
-# nolint start
 #####################################################################################
```

Comment:
please also remove these top header lines, we don't need those anymore (lines 1-14)

Reply:
ok, removed